engine/src/test/resources/org/camunda/bpm/engine/test/bpmn/parse/BpmnParseTest.testParseWithoutStartEvent.bpmn

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I implemented your suggested changes, thanks! However, I see a lot of test files with the .bpmn extension as well. Additionally, the Camunda modeler also seems to create .bpmn files by default. Perhaps something to look at to prevent having to include this review comment in future PRs.

Reply:
I agree that we do not have to be too strict on this and that we already have a mixture of files here. However, thanks for keeping it simple @daveychu and simply adjusting it. 😃